### ISSUE TITLE
wslgit stopped working after upgrading to 0.8.0

### ISSUE THREAD

**Issue by ScarletFlash** [andy-5/wslgit#92]
After switching to https://github.com/andy-5/wslgit/tree/v0.8.0, wslgit stopped working.
Previously I used to use https://github.com/andy-5/wslgit/tree/v0.7.0 (wslgit-mount-at-root.exe). Everything was OK.

**lsb_release -a:**
```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 19.10
Release:        19.10
Codename:       eoan
```

**OS:**
```
Windows 10 Pro 64-bit (10.0, Build 18362) (18362.19h1_release.190318-1202)
```

**Env. variables:**
```
WSLGIT_MOUNT_ROOT: true
WSLGIT_USE_INTERACTIVE_SHELL: smart
```

**Comment by andy-5** [comment]
Unfortunately, the current documentation is a bit unclear. You need to set the new Windows environment variable `WSLGIT_MOUNT_ROOT` to `/` to get the behaviour of `wslgit-mount-at-root.exe`.

Also, `WSLGIT_USE_INTERACTIVE_SHELL=smart` is only available in the upcoming v0.9.0 - the current Readme already reflects the state of the master branch.

Does this fix your problem?

**Comment by ScarletFlash** [comment]
Yes, thank you.
